webapp/app.py

```diff
@@ -190,4 +191,5 @@ def handle_unauthorised(error):
     """
     Handle 401 errors using flask as opposed to requests
     """
+    flask.session.pop("authentication_token", None)
```

Comment:
If I understand this correctly, the user would have to manually go to anbox-demo.io/login to get a new token, right ?
If that's the case we should redirect automatically rather than show a dedicated 401 page

Reply:
Yes that's right

Reply:
Ok, then we need the redirection to /login.
Without it, a user would have to manually trigger the login process every 24h (current duration of auth token)